src/WebJobs.Script/Metrics/HostMetrics.cs

```diff
@@ -57,8 +57,34 @@ public HostMetrics(IMeterFactory meterFactory, IEnvironment environment)
             _startedInvocationCount = meter.CreateCounter<long>(StartedInvocationCount, "numeric", "Number of function invocations that have started.");
         }
 
-        public void AppFailure() => _appFailureCount.Add(1);
+        private KeyValuePair<string, object> FunctionGroupTag
+        {
+            get
+            {
+                if (_cachedFunctionGroupTag != null)
+                {
+                    return _cachedFunctionGroupTag.Value;
+                }
+
+                var functionGroup = _environment.GetEnvironmentVariableOrDefault(EnvironmentSettingNames.FunctionsTargetGroup, string.Empty);
+                var functionGroupTag = new KeyValuePair<string, object>(TelemetryAttributes.AzureFunctionsGroup, functionGroup);
+
+                if (!string.IsNullOrEmpty(functionGroup))
```

Comment:
Should we log when we're unable to resolve a function group? Having telemetry here would really help if this is missing when expected to be present.

Reply:
Done!